cms/templates/manage_users.html

```diff
@@ -114,11 +117,11 @@ <h3 class="title-3">${_("Transferring Ownership")}</h3>
 <%block name="requirejs">
   require(["js/factories/manage_users"], function(ManageCourseUsersFactory) {
       ManageCourseUsersFactory(
-        "${context_course.display_name | h}",
-        ${escape_json_dumps(users) | n},
-        "${reverse('contentstore.views.course_team_handler', kwargs={'course_key_string': unicode(context_course.id), 'email': '@@EMAIL@@'})}",
-        ${ request.user.id },
-        ${str(allow_actions).lower()}
+        "${context_course.display_name_with_default | h}",
+        ${users | n, dump_js_escaped_json},
+        "${reverse('contentstore.views.course_team_handler', kwargs={'course_key_string': unicode(context_course.id), 'email': '@@EMAIL@@'}) | n, js_escaped_string}",
+        ${request.user.id | n, dump_js_escaped_json},
+        ${bool(allow_actions) | n, dump_js_escaped_json}
```

Comment:
It seems like `allow_actions` is already a boolean, but I guess it couldn't hurt.